restservice/src/main/java/org/itevents/controller/ServiceExceptionHandler.java

```diff
@@ -38,11 +41,13 @@
 
     @ExceptionHandler(ActionAlreadyDoneServiceException.class)
     public ResponseEntity<String> handleActionAlreadyDoneServiceException(ActionAlreadyDoneServiceException ex) {
+        LOGGER.error(ex.getMessage());
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.CONFLICT);
     }
 
     @ExceptionHandler(ConstraintViolationException.class)
-    public ResponseEntity<String>  handleConstrainViolationExceptions(ConstraintViolationException ex) {
+    public ResponseEntity<String>  handleConstrainViolationException(ConstraintViolationException ex) {
```

Comment:
1) typo in handleConstrain**t**ViolationException
2) delete redundant space after `<String>`